omegaconf/nodes.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, parent: Optional[Container], value: Any, metadata: Metadata):
 
         super().__init__(parent=parent, metadata=metadata)
         with read_write(self):
-            self._set_value(value)
+            self._set_value(value)  # lgtm [py/init-calls-subclass]
```

Comment:
lgtm is still complaining about it. maybe some syntax issue with the suppression comment?

Reply:
I'm a bit confused by lgtm, it seems like the warning will be there until merged, since they in the warning link they say: "If you make changes to the suppression of alerts in a pull request, please note that theses changes will only be effective once that pull request is merged"
(this sounds dumb, but I guess there must be a reason)